src/beanahead/scripts/cli.py

```diff
@@ -89,6 +90,11 @@ def main():
         choices=["x", "rx", "rx_def"],
         metavar="key",
     )
+    parser_make.add_argument(
```

Comment:
Could you make the 'main_ledger' arg optional, with the implementation assuming default category names if it's not passed. (For backwards compatibility as much as anything else.)

Reply:
Oh, sure! I should have done that! :+1: